user/serializers.py

```diff
@@ -2,0 +1,27 @@
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff")
+        read_only_fields = ("id", "is_staff")
+        extra_kwargs = {
+            "password": {"write_only": True,
+                         "min_length": 5
+                         }
+        }
+
+    def create(self, validated_data):
+        return get_user_model().objects.create_user(**validated_data)
+
+    def update(self, instance, validated_data):
+        password = validated_data.pop("password", None)
```

Comment:
When updating the user, ensure that the `password` field is correctly handled. The `set_password` method is used, which is correct for hashing the password. Ensure that any other sensitive fields are also handled appropriately.